.travis.yml

```diff
@@ -1,13 +1,38 @@
-language: go
+stages:
+  - test
+  - name: publish
+    if: branch = master
```

Comment:
you may want to gate this to the actual master branch as this will also run on PR builds for PRs targeteat the master branch

```
if: branch = master AND TRAVIS_PULL_REQUEST = false
```